server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -206,3 +206,23 @@ body {
 main {
   flex: 1 0 auto;
 }
+
+// Tablet: Program card grid has 2 columns
+@media (min-width: 40em) {
+  .cf-program-card-group {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+// Desktop: Program card grid has 3 columns
+@media (min-width: 64em) {
+  .cf-program-card-group {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+
+.cf-application-card {
```

Comment:
question (maybe blocking): I see the reference class being used in existing client facing code. Will these style updates affect those elements at all?

Reply:
I don't think so, since this file is only applied to North Star, and none of the old screenshots changed when I ran the browser tests. Updated the name anyway.